config/locales/pets.en.yml

```diff
@@ -0,0 +1,33 @@
+en:
+  errors:
+    messages:
+      # messages for active storage validations gem
+      content_type_invalid: "must be PNG or JPEG YOOOO"
+      file_size_out_of_range: "size must be between 10KB and 1MB"
+      limit_out_of_range: "5 maximum"
+  activerecord:
+    errors:
+      models:
+        pet:
+          attributes:
+            name:
+              blank: "can't be blank"
```

Comment:
if preferred I can refer all of these repetitive messages to a single messages key that is this value